www/_template/getting-started.md

```diff
@@ -3,43 +3,278 @@ layout: layouts/main.njk
+
+IMAGE: Gif showing the code next to the project running in the browser. On save the console shows "Hello World!". On edit and save of the `hello-world.js` file to be "Hello everyone!" instead, that instantly shows in the browser console.
+
+## Using npm Packages
+
+While writing your own JavaScript is great, you'll also probably want to use JavaScript from npm. npm packages are mainly published using a module syntax (Common.js, or CJS) that can't run on the web without some build processing. Even if you write your application using browser-native ESM `import` and `export` statements that would all run directly in the browser, trying to import any one npm package will force you back into bundled development.
+
+**Snowpack takes a different approach:** instead of bundling your entire application for this one requirement, Snowpack processes your dependencies separately. Here's how it works:
+
+```
+node_modules/react/**/*     -> http://localhost:3000/web_modules/react.js
+node_modules/react-dom/**/* -> http://localhost:3000/web_modules/react-dom.js
+```
+
+1. Snowpack scans your website/application for all used npm packages.
+2. Snowpack reads these installed dependencies from your `node_modules` directory.
+3. Snowpack bundles all of your dependencies separately into single JavaScript files. For example: `react` and `react-dom` convert to `react.js` and `react-dom.js`, respectively.
+4. Each resulting file runs directly in the browser, and imported via ESM `import` statements.
+5. Because your dependencies rarely change, Snowpack rarely needs to rebuild them.
+
+After Snowpack builds your dependencies, any package can be imported and run directly in the browser with zero additional bundling or tooling required. This ability to import npm packages natively in the browser (without a bundler) is the foundation that all unbundled development and the rest of Snowpack builds on top of.
+
+Snowpack lets you import npm packages directly in the browser. Even if a package is using a legacy format, Snowpack will up-convert the package to ESM before serving it to the browser.
+
+> 💡 Tip: when you start up your development server or run a new build, you may see a message that Snowpack is "installing dependencies." This means that Snowpack is converting your dependencies to run in the browser.
+
+Install the canavas-confetti package from npm and use it with the following command:
 
 ```bash
-# Start your dev server, load your site locally
-snowpack dev
+npm install --save canvas-confetti
+```
+
+Now head to `index.html` and add this code to the bottom of the `<body>` tag:
+
+```diff
+   <script type="module" src="/index.js"></script>
++  <script type="module">
```

Comment:
Lets fix the bug regardless, but +1 to adding this code into your index.js file instead. 

I always really want to make sure that users understand that Snowpack can feel exactly like what they're used to, and writing scripts inside of HTML is a bit of a unique thing that we support. 

I like the idea of adding this as a Tip though, something like "Did you know, with Snowpack you can also add this code directly to your HTML if you prefer!"